Sources/Orbit/Support/BindingSource.swift

```diff
@@ -6,6 +6,8 @@ public enum OptionalBindingSource<Value> {
     case state(Value)
 }
 
+extension Binding: Sendable where Value: Sendable {}
```

Comment:
Why is this necessary? `Binding` already conforms to `Sendable`. Is this for some older OSes?

Reply:
Good point. This plus below were errors in my previous attempt to fix snapshot testing builds. It may not be necessary anymore, will check...

Reply:
Removed